### PR TITLE
Feat/react select dropdown

### PR DESCRIPTION
**Pull request by JIamep20**
1. installed react-select
2. implemented "Select all" feature
3. Wrapped it into "react-hook-form" controller
4. Styled a little to follow design in figma

### CODE REVIEW

File: src/components/Select/Select.js [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+  singleValue: (styles) => ({
+    ...styles,
+    fontSize: "16px",
+    lineHeight: "24px"
+  })
+};
+
+export const Select = ({ name, control, ...props }) => {
+  return (
+    <Controller
```
Comment (moshloop) [inline src/components/Select/Select.js:150]: The `control` is optional you can specify either control or onSelect

**Comment by moshloop** [comment]
Also there is this border: 

Screenshot 2022-02-04 at 13 33 42

File: src/components/Select/Select.js (hunk 1 above)
Comment (JIamep20) [reply src/components/Select/Select.js:150]: What's the point in additional onSelect props? You can use original onChange prop... Or you want it to transform somehow value.

Comment (moshloop) [reply src/components/Select/Select.js:150]: either onSelect or onChange is fine - by control must be optional

Comment (JIamep20) [reply src/components/Select/Select.js]: done, removed border in the input

**Comment by moshloop** [comment]
@JIamep20  please can you resolve conflicts and squash your commits ?

**Pull request merged**